### DISCUSSION TITLE
Support Django `path` in ViewSet decorator `action`

### DISCUSSION THREAD

**Discussion by amchii** [encode/django-rest-framework#8126] [Ideas & Suggestions] [answered]
Django's `path` is convenient to use, and drf's `url_path`  is similar to Django's `re_path` that uses regex. So may drf add a parameter `path`(or `url_path`?) to support pattern likes `<int: year>` and a parameter `url_re_path` uses regex to ViewSet decorator `action` ?
Like this:
```python

class UserViewSet(viewsets.ViewSet):
    @action(methods=['GET'], detail=True, url_path="collections/<int:collection_id>/blogs")
    # @action(methods=['GET'], detail=True, url_path="collections/(?P<collection_id>[0-9]+)/blogs")
    def get_collected_blogs(self, request, pk=None, collection_id=None, **kwargs):
        return Response(status=status.HTTP_200_OK)

```

**Comment by browniebroke** [comment, marked as answer]
I think that was added in https://github.com/encode/django-rest-framework/pull/6789 (released in DRF 3.15 earlier this year)

You need to set it at the router level:

```diff
- my_router = SimpleRouter()
+ my_router = SimpleRouter(use_regex_path=False)
```
